pages/datasets/[did].vue

```diff
@@ -329,11 +337,60 @@
           :dataset
         />
 
+        <div
+          v-if="dataset.access_type === 'restricted'"
+          class="container"
+        >
+          <SimpleBanner
+            type="pink"
+            class="p-4 flex justify-between items-center"
+          >
+            <div class="space-y-3.5">
+              <div>
+                <AdminBadge
+                  :icon="RiLockLine"
+                  size="xs"
+                  type="pink"
+                >
+                  {{ $t('Accès restreint') }}
+                </AdminBadge>
+              </div>
+              <p class="font-bold text-xl">
+                {{ $t('Ces données ne sont accessibles que sur habilitation') }}
+              </p>
+              <p
+                v-if="dataset.access_type_reason"
```

Comment:
Do we want to show the reason category if we only have it ?

Reply:
Fixed in f4ef0de